src/SFA.DAS.PR.Domain/Entities/AccountLegalEntity.cs

```diff
@@ -1,5 +1,8 @@
-﻿namespace SFA.DAS.PR.Domain.Entities;
+﻿using System.Diagnostics.CodeAnalysis;
 
+namespace SFA.DAS.PR.Domain.Entities;
+
+[ExcludeFromCodeCoverage]
```

Comment:
I have never marked entities with exclusion from coverage as they do not have any logic, was this being reported as no coverage?